client/README.md

```diff
@@ -6,11 +6,51 @@ This extension provides a Yseop Markup Language (YML) basic support for VS Code.
 
 - Syntax coloring for classic code
 - Syntax coloring for text granules and inside text granules
-- Snippets:
+- Snippets (`prefix`: Description):
+    * `\amount`: Display amount
```

Comment:
I think that with a French keyboard, writing `\` character is very annoying. That's why we added the “granule” snippet.
Maybe we can think of an easier to write character at the begining? Like `tagamount` or `tamout` (`t` for `tag`).

Reply:
The idea is that people who do not know about some snippets may still stumble on it while typing their code exactly as usual, hence the prefixes with the `\`. Dude, I spent whole days writing LaTeX with even more backslashes than that and it never really struck me as being *that* hard. I think the `\( \)` combination for granules is an exceptional case, though, as it forces the hands to adopt a rather unusual position together.

Reply:
Another interesting thing is that just typing `am` would suggest boatloads of things from the predefined objects, while `\a` is far more restrictive and gives useful stuff.